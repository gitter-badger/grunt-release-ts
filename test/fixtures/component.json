{
  "name": "grunt-release-ts-test",
  "version": "0.0.12"
}
