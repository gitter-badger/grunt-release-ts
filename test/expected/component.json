{
  "name": "grunt-release-ts-test",
  "version": "0.0.13"
}
